test/box-py/args.result

```diff
@@ -1,41 +1,81 @@
 tarantool --help
-Tarantool - a Lua application server
-
-Usage: tarantool script.lua [OPTIONS] [SCRIPT [ARGS]]
-
-All command line options are passed to the interpreted script.
-When no script name is provided, the server responds to:
-  -h, --help			display this help and exit
-  -v, --version			print program version and exit
-  -e EXPR			execute string 'EXPR'
-  -l NAME			require library 'NAME'
-  -j cmd			perform LuaJIT control command
-  -b ...			save or list bytecode
-  -d				activate debugging session for 'SCRIPT'
-  -i				enter interactive mode after executing 'SCRIPT'
-  --				stop handling options
-  -				execute stdin and stop handling options
+Tarantool Server 3.<minor>.<patch>-<suffix>
```

Comment:
Why "Tarantool Server" and not "Tarantool"? Is it a requirement from a product team?

Reply:
It was originally proposed by @Mons, but we've finally decided to omit this to make in consistent with the interactive console prompt.

Reply:
Fixed.